pkg/clusterapi/proxy.go

```diff
@@ -0,0 +1,91 @@
+package clusterapi
+
+import (
+	_ "embed"
+	"fmt"
+
+	bootstrapv1 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1beta1"
+	controlplanev1 "sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1beta1"
+
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/templater"
+)
+
+//go:embed config/http-proxy.conf
+var proxyConfig string
+
+var NoProxyDefaults = []string{
```

Comment:
Does this need to be public? If it is, I would vote to expose it through a method. This makes the variable mutable from the outside of the package